DQMServices/Components/plugins/DQMGenericClient.cc

```diff
@@ -157,13 +157,15 @@ class FitSlicesYTool {
     TH1::AddDirectory(true);
     // ... create your hists
     TH2F* h = me->getTH2F();
-    h->FitSlicesY(nullptr, 0, -1, 0, "QNR SERIAL");
+    TF1* fitFcn = new TF1("gaus", "gaus");
```

Comment:
I was checking the code of `TH2::FitSlicesY` and `DoFitSlices` in https://root.cern.ch/doc/master/TH2_8cxx_source.html
I think you should use
```suggestion
    TF1* fitFcn = new TF1("gaus", "gaus",h->GetYaxis().GetXmin(),h->GetYaxis().GetXmax());
```
Specifically, please look at this line https://root.cern.ch/doc/master/TH2_8cxx_source.html#l00761